python/test/t_FunctionalChaosResultPlotSensitivity_std.py

```
@@ -0,0 +1,49 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+import openturns as ot
+from openturns.viewer import PlotSensitivity
+
+ot.TESTPREAMBLE()
+ot.RandomGenerator.SetSeed(0)
+
+try:
```

these try blocks can mask the error if it went to fail